assets/styles/cookie-consent.scss

```diff
@@ -192,21 +161,72 @@
     text-align: center;
     vertical-align: top;
     color: #555;
-    font-size: 18px;
+    font-size: 20px;
+  }
+
+  &:checked {
+
+    & + .cookie-consent-option__label {
+      font-weight: normal;
+
+      &:before {
+        padding: 2px;
+        content: '\2713';
+        color: var(--cookie-notice-checkbox-color);
+      }
+    }
+
+    &:disabled + .cookie-consent-option__label:before {
+      border: 2px solid currentColor;
```

Comment:
What `currentColor` is? 🤔

Reply:
The colour of the current element - https://caniuse.com/#feat=currentcolor

Very useful 😉